js/package.json

```diff
@@ -54,7 +54,7 @@
   },
   "jupyterlab": {
     "extension": "src/jupyterlab-plugin",
-    "outputDir": "../share/jupyter/labextensions/jupyter-threejs",
+    "outputDir": "../pythreejs/labextension",
```

Comment:
I don't think this makes things simpler to understand, having the same directory keeps things simple.

Reply:
OK, will revert this :)

Reply:
the only concern might be path length, if it's getting deep, for windows folks. 256 chars comes up so fast... and frigging CI machines already have it set, so it's up to users to discover...

Reply:
I'm not a windows user these days, but I can appreciate the concern; @maartenbreddels do you feel like comfortable leaving it in-tree?

Reply:
ok, lets leave it as is.